### ISSUE TITLE
How to install the latest version on Windows

### ISSUE THREAD

**Issue by ericop** [kika/fixedsys#15]
# How do I install the latest version of the `.ttf` on Windows?

This font is super cool, and retro-tastic!

The problem is I'd love to get the most updated [(v3.02.9)](https://github.com/kika/fixedsys/tree/v3.02.9) ligatures from something like http://www.fixedsysexcelsior.com/fonts/lastest.ttf or have clear instruction on how to install them in them in the Readme.  (It was a but fuzzy for me if i had to install python and then this 3rd party tool, https://github.com/fonttools/fonttools, etc. )

**You've got a cool thing here**, and I'd hate to have people miss out on it because the barrier to entry is kind of high. Can you dumb it down for us consumer types? 😉

**Comment by iiiGerardoiii** [comment]
You can get the latest version here: https://github.com/kika/fixedsys/releases/latest, download FSEX302.ttf

I'm not using Windows atm, but if I recall correctly you just have to download the .ttf file, double-click it and hit Install.

**Comment by ericop** [comment]
@iiiGerardoiii Thanks for pointing out the obvious, seriously. 😃  I downloaded the zip of the souce and dug around only finding the `.ttx` file and was confused as to why the `.ttf` wasn't in there, but I overlook it on that release as it's own file. 

So yes, this works as expected, just a double click and away you go.
